qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -779,3 +659,211 @@ void HOCRPdfExporter::updateValid() {
 	}
 	ui.buttonBox->button(QDialogButtonBox::Ok)->setEnabled(valid);
 }
+
+void HOCRPdfExporter::validExportOptionChanged(int index)
+{
+    bool valid = true;
+    if(index == 1)
+        valid = false;
+    ui.groupBoxEncryption->setEnabled(valid);
+    ui.comboBoxImageCompression->setEnabled(valid);
+    ui.comboBoxImageFormat->setEnabled(valid);
+    ui.lineEditAuthor->setEnabled(valid);
+    ui.lineEditTitle->setEnabled(valid);
+    ui.lineEditProducer->setEnabled(valid);
+    ui.lineEditKeywords->setEnabled(valid);
+    ui.lineEditSubject->setEnabled(valid);
+    ui.lineEditKeywords->setEnabled(valid);
```

Comment:
You should be able to just disable the entire advanced tab, since none of the entries there is applicable to the QPainter export.

Reply:
QPainter supports **Creator** option

Reply:
Ah ok